users/goelashwin36/Task-3/js/javascript.js

```diff
@@ -0,0 +1,10 @@
+function check(){
+    var radioValue = 0;
+    for (var i =1; i <=5; i++){
+        selector = $("input[name = '" + i + "']:checked");
+        radioValue=radioValue+ Number(selector.val());
+    }
+    $("#p1").text("Your score is- "+radioValue+"/5");
```

Comment:
This gives me `Nan/5` when I don't select any on of the options, it should rather show `0/5` even if none of the options are selected. Fix this